_posts/en/newsletters/2019-12-04-newsletter.md

```diff
@@ -0,0 +1,154 @@
+---
+title: 'Bitcoin Optech Newsletter #75'
+permalink: /en/newsletters/2019/12/04/
+name: 2019-12-04-newsletter
+slug: 2019-12-04-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes some recent discussion about the
+schnorr and taproot proposals, notes the recent name change of the
+proposal formerly known as `OP_CHECKOUTPUTSHASHVERIFY` and
+`OP_SECURETHEBAG`, and summarizes notable changes to popular Bitcoin
+infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Continued schnorr/taproot discussion:** this week, Russell O'Connor
+  started a [thread][oconnor checksig pos] on the Bitcoin-Dev mailing list continuing a
+  [previous discussion][wuille safer sighashes] about having signatures commit to the position
+  of the opcode that's expected to evaluate that signature (e.g.
+  the opcodes `OP_CHECKSIG`, `OP_CHECKSIGVERIFY`, or tapscript's new
+  `OP_CHECKSIGADD`).  O'Connor argues that this provides additional
+  protection for people who use scripts with multiple branches that
+  allow the script to be satisfied in several different ways using
+  signatures from more than one user.  Without this protection, it might
+  be possible for Mallory to tell Bob that he's signing for branch *A*
```

Comment:
FYI: Mallory is a she: https://nameberry.com/babyname/Mallory (the ability to use simple gendered pronouns is why I tend to use Bob and Mallory rather than Alice and Mallory).  I do think changing the verb to "ask" is an improvement, so I'll do that.

Reply:
Interesting. I suppose I'm more familiar with Mallory as a surname. You're right that Mallory as a given name is more common for women, but the Wikipedia disambiguation page has at least three men with the given name Mallory.

In any case, I've never met a Mallory, and I suspect the same is true for many of our readers, so removing the (potentially) ambiguous pronoun is an improvement.